components/tests/ui/testcases/web/delete_test.txt

```diff
@@ -74,19 +80,20 @@ Test Delete Images in Dataset
     Click Dialog Button                     Yes
     # Should see almost instant removal of 1 thumbnail...
     ${delThumbCount}=                       Evaluate   ${thumbCount} - 1
-    Wait Until Keyword Succeeds             1   0.1   Xpath Should Match X Times   ${thumbnailsXpath}   ${delThumbCount}
-    # ...Need to check that centre panel doesn't reload and show image during delete: #12866
-    Sleep                                   5
+    Wait Until Keyword Succeeds     ${TIMEOUT}      ${INTERVAL}   Xpath Should Match X Times   ${thumbnailsXpath}   ${delThumbCount}
+    # Wait for activities to show job done, then refresh tree...
+    Wait Until Keyword Succeeds     ${TIMEOUT}   ${INTERVAL}   Page Should Contain Element     xpath=//span[@id='jobstatus'][contains(text(),'1')]
+    Click Element                           refreshButton
+    Wait Until Page Contains Element        xpath=//li[@rel='experimenter']//a[contains(@class, 'jstree-clicked')]  ${WAIT}
+    Wait Until Page Contains Element        id=dataIcons    ${WAIT}
```

Comment:
See https://github.com/openmicroscopy/openmicroscopy/pull/3792 and ticket (12866) to see if the bug-fix (first commit of the PR) is still caught by this test.